[inductor][test] remove no-op test

nmacchioni opened the pull request:
Summary: this test isn't meaningful, since force_shape_pad is set

Test Plan: CI

Differential Revision: D88388188


@diff-train-skip-merge


cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @kadeng @muchulee8 @amjames @chauhang @aakhundov @coconutruben @jataylo

v0i0 commented:
i think this test is still meaningful, since it tests https://github.com/pytorch/pytorch/blob/main/torch/_inductor/fx_passes/pad_mm.py#L422-L440 ?

izaitsevfb commented:
@pytorchbot merge

izaitsevfb commented:
ok, this was also reverted internally, I shouldn't have tried to merge it

nmacchioni committed:
[inductor][test] remove no-op test (#169580)

Summary:

this test isn't meaningful, since force_shape_pad is set

Test Plan: CI

Reviewed By: v0i0

Differential Revision: D88388188
